src/transformers/pdfmakeToPdf.ts

```diff
@@ -0,0 +1,20 @@
+import * as pdfmake from 'pdfmake/build/pdfmake'
+import { File } from '../manager'
+import renderPDF from '../utils/renderPDF'
+
+export const pdfmakeMimeType = 'x-application/pdfmake'
+
+export default async function pdfmakeToPdf(input: File): Promise<File> {
+  if (input.contentType !== pdfmakeMimeType) {
+    return input
```

Comment:
I'm a tad confused as to why this design choice is good -- letting input stay unmodified without any errors when the mime type doesn't work. Is there a good reason not to fail hard?

Reply:
These transforms are best thought of as a pipeline. They receive all inputs in order, and can choose to modify it or not. They choose what to modify based on the content type.